vlib/v/checker/tests/mul_op_wrong_type_err.out

```diff
@@ -1,53 +1,53 @@
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:5:13: error: mismatched types `Aaa` and `any_int`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:5:13: error: mismatched types `Aaa` and `untyped int`
     3 | struct Aaa{}
     4 | fn main() {
     5 |     println(Aaa{} * 10)
       |             ~~~~~
     6 |     println(10 * Aaa{})
     7 |     println([1,2,3] * 10)
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:6:18: error: mismatched types `any_int` and `Aaa`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:6:18: error: mismatched types `untyped int` and `Aaa`
     4 | fn main() {
     5 |     println(Aaa{} * 10)
     6 |     println(10 * Aaa{})
       |                  ~~~~~
     7 |     println([1,2,3] * 10)
     8 |     println(10 * [1,2,3])
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:7:13: error: mismatched types `[]int` and `any_int`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:7:13: error: mismatched types `[]int` and `untyped int`
     5 |     println(Aaa{} * 10)
     6 |     println(10 * Aaa{})
     7 |     println([1,2,3] * 10)
       |             ~~~~~~~
     8 |     println(10 * [1,2,3])
     9 |     a := map[string]int
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:8:18: error: mismatched types `any_int` and `[]int`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:8:18: error: mismatched types `untyped int` and `[]int`
     6 |     println(10 * Aaa{})
     7 |     println([1,2,3] * 10)
     8 |     println(10 * [1,2,3])
       |                  ~~~~~~~
     9 |     a := map[string]int
    10 |     println(a * 10)
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:10:13: error: mismatched types `map[string]int` and `any_int`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:10:13: error: mismatched types `map[string]int` and `untyped int`
     8 |     println(10 * [1,2,3])
     9 |     a := map[string]int
    10 |     println(a * 10)
       |             ^
    11 |     println(10 * a)
    12 |     c1 := cmplx.complex(1,-2)
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:11:18: error: mismatched types `any_int` and `map[string]int`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:11:18: error: mismatched types `untyped int` and `map[string]int`
     9 |     a := map[string]int
    10 |     println(a * 10)
    11 |     println(10 * a)
       |                  ^
    12 |     c1 := cmplx.complex(1,-2)
    13 |     c2 := c1 * 2.0
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:13:11: error: infix expr: cannot use `any_float` (right expression) as `math.complex.Complex`
+vlib/v/checker/tests/mul_op_wrong_type_err.vv:13:11: error: infix expr: cannot use `untyped float` (right expression) as `complex.Complex`
    11 |     println(10 * a)
    12 |     c1 := cmplx.complex(1,-2)
    13 |     c2 := c1 * 2.0
       |              ^
    14 |     println(c2)
    15 |     c3 := 2.0 * c1
-vlib/v/checker/tests/mul_op_wrong_type_err.vv:15:12: error: infix expr: cannot use `math.complex.Complex` (right expression) as `any_float`
```

Comment:
I don't like how in these cases type_to_str doesn't show the full type name, I see it was an option there in how it works. what do you think @spytheman. I know it was done like this for its use in doc/fmt which makes sense. but in this case it docent make much sense

Reply:
imho: the short version makes more sense because it's the same how it is in the code. The user isn't confused then. But i think it's a personal preference thing.

Reply:
when looking at the error though you don't know what the imports look like it could be a type that's not distinguishable

Reply:
ah, i know what you mean.

Reply:
Do you know how to fix it in this case?

Reply:
Yeah this can be fixed